### ISSUE TITLE
Added Bug  in commit ea25475

### ISSUE THREAD

**Issue by manuelj555** [formapro/JsFormValidatorBundle#102]
The commit https://github.com/formapro/JsFormValidatorBundle/commit/ea25475f1a2a5382325254e0167bc4619e256f99 remove the `Resources/public/js/fp_js_validator.js` file, and this file is used in https://github.com/formapro/JsFormValidatorBundle/blob/master/Resources/views/javascripts.html.twig twig file for loading the fp_js_validator library.

**Comment by 66Ton99** [comment]
Yes, but it is not bug!
Resources/public/js/fp_js_validator.js - generates automatically by assets http://symfony.com/doc/current/cookbook/assetic/asset_management.html

**Comment by manuelj555** [comment]
Are you sure that these file is generated by assetic?

The twig file `FpJsFormValidatorBundle::javascripts.html.twig` import the `fp_js_validator.js` using the asset function:

```twig
<script src="{{ asset('bundles/fpjsformvalidator/js/fp_js_validator.js') }}" type="text/javascript"></script>
{{ js_validator_config() }}
{{ init_js_validation() }}
```
I am executing the command `php app/console assetic:dump` and the `bundles/fpjsformvalidator/js/fp_js_validator.js` is not generated.

**Comment by 66Ton99** [comment]
Yes
It is new setup of Symfony 2.4
```
ls -l vendor/fp/jsformvalidator-bundle/Fp/JsFormValidatorBundle/Resources/public/js/
total 208
-rw-r--r--  1 66Ton99  specialm  30345 May 15  2015 FpJsFormValidator.js
drwxr-xr-x  2 66Ton99  specialm   1020 May 15  2015 constraints
-rwxr-xr-x  1 66Ton99  specialm  65851 May 15  2015 fp_js_validator.js
-rw-r--r--  1 66Ton99  specialm    732 May 15  2015 jquery.fpjsformvalidator.js
drwxr-xr-x  2 66Ton99  specialm    374 May 15  2015 transformers
```

**Comment by manuelj555** [comment]
Mmm how could assetic create assets in vendors directories?

In my PC does not exists the `fp_js_validator.js` before or after execute `assetic:dump and assets:install`:

    ls -l vendor/fp/jsformvalidator-bundle/Fp/JsFormValidatorBundle/Resources/public/js/
    total 20
    -rw-r--r--    1 Manuel A Administ    30345 Mar  4 14:10 FpJsFormValidator.js
    drwxr-xr-x   30 Manuel A Administ     4096 Mar  4 14:10 constraints
    -rw-r--r--    1 Manuel A Administ      732 Mar  4 14:10 jquery.fpjsformvalidator.js
    drwxr-xr-x   11 Manuel A Administ     4096 Mar  4 14:10 transformers

**Comment by 66Ton99** [comment]
I got your problem and did not able to solve it. But you can look on travis build script and it works there without problem.
fp_js_validator.js - contains compiled code from `FpJsFormValidator.js` and `transformers/*`

**Comment by 66Ton99** [comment]
it will be fixed here #109
